packages/database/drizzle/0032_worthless_doctor_octopus.sql

```
@@ -0,0 +1,64 @@
+CREATE UNIQUE INDEX "team_members_team_user_uniq" ON "team_members" USING btree ("team_id","user_id");--> statement-breakpoint
+CREATE INDEX "team_members_user_idx" ON "team_members" USING btree ("user_id");--> statement-breakpoint
+CREATE INDEX "teams_organization_idx" ON "teams" USING btree ("organization_id");--> statement-breakpoint
+ALTER TABLE "sessions" ADD CONSTRAINT "sessions_active_organization_id_organizations_id_fk" FOREIGN KEY ("active_organization_id") REFERENCES "public"."organizations"("id") ON DELETE set null ON UPDATE no action;--> statement-breakpoint
+ALTER TABLE "profiles" DROP COLUMN "organization_id";--> statement-breakpoint
+ALTER TABLE "organizations" ADD CONSTRAINT "organizations_slug_unique" UNIQUE("slug");--> statement-breakpoint
+-- Clean up legacy institution organizations: pre-Better-Auth rows created from the
+-- registration free-text "organization" field. They have no slug and no members, so
+-- they are invisible to the organization plugin and safe to remove.
+DELETE FROM "organizations" o
+WHERE o."slug" IS NULL
+  AND NOT EXISTS (SELECT 1 FROM "organization_members" m WHERE m."organization_id" = o."id");
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)